pubsub/README.rst

```diff
@@ -41,6 +39,88 @@ Here's the rough usage pattern for using ``SubscriberClient``:
         max_messages=10)
 
 
+There's also ``gcloud.aio.pubsub.subscribe`` helper function you can use to
+setup a pubsub processing pipeline. It is built with ``asyncio`` and thus only available
+in ``gcloud-aio-pubsub`` package. The usage is fairly simple:
+
+.. code-block:: python
+
+    from gcloud.aio.pubsub import SubscriberClient
+    from gcloud.aio.pubsub import subscribe
+    from gcloud.aio.pubsub.metrics_agent import MetricsAgent
+
+    subscriber_client = SubscriberClient()
+
+    async def handler(message):
+        return
+
+    await subscribe(
+        'projects/<my_project>/subscriptions/<my_subscription>',
+        handler,
+        subscriber_client,
+        num_producers=1,
+        max_messages_per_producer=100,
+        ack_window=0.3,
+        num_tasks_per_consumer=1,
+        metrics_client=MetricsAgent()
+    )
+
+While defaults are somewhat sensible, it is highly recommended to performance test
+your application and tweak function parameter to your specific needs. Here's a few hints:
+
+:``handler``:
+    an async function that will be called for each message. It should accept an instance of
+    ``SubscriberMessage`` as its only argument and return ``None`` if the message should
+    be acked.
+
+:``num_producers``:
+    number of workers that will be making ``pull`` requests to pubsub.
+    Please note that a worker will only fetch new batch once the ``handler``
+    was called for each message from the previous batch. This means that running only a single worker
+    will most likely make your application IO bound. If you notice this being an issue don't hesitate
+    to bump this parameter.
+
+:``max_messages_per_producer``:
+    number of pubsub messages a worker will try to fetch in a single batch. This
```

Comment:
The line width throughout this readme is inconsistent. Should it be 80 chars throughout?

Reply:
Fair enough, time to get a wrap plugin